hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -209,7 +208,13 @@ protected void initSparkContexts(String appName) {
         }))
         .config(jsc.getConf())
         .getOrCreate();
+
     sqlContext = new SQLContext(sparkSession);
+    context = new HoodieSparkEngineContext(jsc, sqlContext);
```

Comment:
This change actually fixes the problem that was previously entailing that `HoodieSparkSessionExtensions` would not be properly injected into created Spark session, b/c session creation was hijacked in `HoodieSparkEngineContext` ctor inadvertently pre-empting the creation thereof by invoking `SQLContext.getOrCreate(jsc.sc())`